watertap/unit_models/zero_order/magprex_zo.py

```diff
@@ -0,0 +1,75 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a Magprex reactor unit
+for struvite precipitation.
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var, Expression
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_sido_reactive, constant_intensity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Marcus Holly"
+
+
+@declare_process_block_class("MagprexZO")
+class MagprexZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a Magprex reactor unit.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "magprex"
+
+        build_sido_reactive(self)
+        constant_intensity(self)
+
+        self.magnesium_chloride_dosage = Var(
+            units=pyunits.dimensionless,
```

Comment:
Why is this dimensionless? What units does it represent?

Reply:
I see-  kg-MgCl2/kg struvite

Reply:
I based it off the electrochemical_nutrient_removal unit model: https://github.com/watertap-org/watertap/blob/main/watertap/unit_models/zero_order/electrochemical_nutrient_removal_zo.py